lib/CurrencyExchangeRateFields/CurrencyExchangeRateFields.js

```diff
@@ -1,12 +1,16 @@
-import React, { useState, useCallback, useEffect } from 'react';
-import { FormattedMessage } from 'react-intl';
-import PropTypes from 'prop-types';
 import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import {
```

Comment:
it's the same question as I raised in another PR, why do you make unnecessary changes (it adds extra lines to review)? especially when we always keep react related at the top imports